src/Context/FixtureServiceAwareContextInterface.php

```diff
@@ -13,8 +24,6 @@
      * Set the FixtureService.
      *
      * @param FixtureService $service
-     *
-     * @return mixed
      */
     public function setFixtureService(FixtureService $service);
```

Comment:
can we use modern `:void` here at some other places, since it goes to new major version and 7.1 as lowest? Adding return typehint is not a bc-break on implementation side, so support both version would be easy

Reply:
Done for almost all methods that do not return any value